pkg/sql/opt/exec/execbuilder/tests/5node/5node_test: TestExecBuild_dist_vectorize failed

pkg/sql/opt/exec/execbuilder/tests/5node/5node_test.TestExecBuild_dist_vectorize [failed](https://teamcity.cockroachdb.com/buildConfiguration/Cockroach_Ci_TestsIbmCloudLinuxS390x_UnitTestsS390x/20934898?buildTab=log) with [artifacts](https://teamcity.cockroachdb.com/buildConfiguration/Cockroach_Ci_TestsIbmCloudLinuxS390x_UnitTestsS390x/20934898?buildTab=artifacts#/) on master @ [78e4507739d52fa371783a14934302e48be2eb01](https://github.com/cockroachdb/cockroach/commits/78e4507739d52fa371783a14934302e48be2eb01):


```
            
        but found (query options: "") :
            planning time: 10µs
            execution time: 100µs
            distribution: <hidden>
            plan type: custom
            rows decoded from KV: 10 (80 B, 20 KVs, 10 gRPC calls)
            DistSQL network usage: <hidden>
            regions: <hidden>
            ·
            • merge join
            │ sql nodes: <hidden>
            │ regions: <hidden>
            │ execution time: 0µs
            │ actual row count: 5
            │ equality: (k) = (k)
            │ left cols are key
            │ right cols are key
            │
            ├── • scan
            │     sql nodes: <hidden>
            │     kv nodes: <hidden>
            │     regions: <hidden>
            │     KV time: 0µs
            │     KV rows decoded: 5
            │     actual row count: 5
            │     missing stats
            │     table: kv@kv_pkey
            │     spans: FULL SCAN
            │
            └── • scan
                  sql nodes: <hidden>
                  kv nodes: <hidden>
                  regions: <hidden>
                  KV time: 0µs
                  KV rows decoded: 5
                  actual row count: 5
                  missing stats
                  table: kw@kw_pkey
                  spans: FULL SCAN
            ·
            Diagram: https://cockroachdb.github.io/distsqlplan/decode.html#eJzsmt9u2sgXx-9_TzGaq_ZXUzz-Q4KlSmjTrpTuJqnSqjerqJrYJ2DZeKhngLBRHmtfYJ9sZRwnAWPKsYhNMFxEsT2253wO5zvfGeaOyp8hdejXT39-OvlG_k9-v7w4I8GEfL44PSfBlFyck2DyPiAfSDB9H1CNRsKDcz4ESZ2_KKMaNahGTapRi2rUplcaHcXCBSlFnDS5m99w6t1SR9eoH43GKjl9pVFXxECdO6p8FQJ16Dd-HcIlcA_itk416oHifjh_TTDpBZMfowBmVKMnIhwPI-mQQCMTqtGvI54ctdpJR_74TpL3SYdELD2Moe-LpIECqdJTyh-CQ_R__5EPTcRUEg9c4YHnkIf7rmcKJImBew45Jr-lJ_uXX06Iy8NQPrYbcT_O2iU9OPt-ckKkghFxxThS5A3cqrYfqbcO0edxpQ0AgqIGQ35LhjAU8YzwMBQuV0m39HkfrrlyByCJGKvRWDkkaT_vfnaC0at7jaZHKf-M7_WMDLgcLJLtJe2vNCoV7wN12LPsnX6kjn6vlUtgZymB014wXZXA6VMC2-yQwi2k0FhKYacwhU_PFbEHMXjLz32XdGSjViu-DWcQ9-Gz8COI22ypnkO4UW967N3bD7HfH6T_Uo1eJNH3mNYztJ6p9RJBgVtwx8oX0WLG1_FNrsmfIVEwHBHPlwEZS96H0vif0JpLaFlxeTxxGEeruK1Edi5aYtRm9iKsAgS_jMMujMNajsNeiINtXuYMr9Nto9U2D2W-baVmZVN4VEKpDyl8CaU-KkxhhUrNXpFSm5sr9WJ5GJuXh1FC4cxW2zqUx7YVziibwuMSCndI4Uso3HFhCitUOOMVKZy-ucItloe5eXmYJRTOarXtQ3lsW-HMsinsllC4QwpfQuG6hSmsUOHMV6RwiNn2YnlYm5eHVULh7NahOLatb1bZBNp4fWu1GeGRRxgRagDxIZnbVjq7MJkVKp31ipQO4eUszLL7JciRiCQsrS-ufpW-9KoWSxiD14c0c1KMYxe-xMKdt00PL-YPmtezB1KlV4304DTKLknF1boFyxdmm3QgBj58_Nli81A6haFEoKYiDkjIFUTu7DGW7PyU-2oxSg8kxD4P_b95HkF227zvMbjgTzJZyS5l2pJdS_UluzoEmXB53kDfsPSf82FIQN2aAelrAOlbAbT0BTKQgFhxNewpIRNLyG4aIStHiCEE1dgpQe3gQukUhrKvgooE1K0ZUPWCigTEiqthTwmZWEJ20whZOULGMiH9OSEzP-Jkj7Jz2myufRRbVDR9p8Q5F4tV0rjv6feG4fgUzwbqTzVDxtI8444EVLtxr9yqmFhCzTPuOUJ2SeO-p4COcHyKZwM7IKjIWJpn3JGAajfu1QsqllDzjHuOUAdr3CsFZK4BZFi_IsQ23Lq2MB_JDTlHO-3hKy8yA8en7hX96uc4SEDFE4P6x2QDGUvtHr76tSIsoeZ5-Byh45328JUL6jGOT90r-tULKhJQ8cRgBwQVGUvtHr56QcUSap6HzxHq7rSHrxyQnRtxWNkNNHtKyEQCqtvEV_8rMpZQ85b3sYTWzA3qH5dNdDTN8_F5RGV30ewpoS4SUN1OvnpVxRJq3ho_ltCaCcIOqCo6muaZ-Twi9Faa_RYNOz_u5HYINdvNW0hAzXPzWELNc_NYQg1c6UcjKp4i1D80W_loMHsVGzAy20hAzXPzWELNc_NYQg1c7kcjKp4i7ICq5qPJ7UDaKTdfvapa91cavQnF9IfvUYfqD5_Wij_ZhyY38L6kzh39OhDTOa1vsxFI6tzwUIJGz3gAH0FBPPQjXyrfpY6Kx3B__7__AgAA__-nAaOb
            
[15:46:32] --- done: /home/roach/.cache/bazel/_bazel_roach/c5a4e7d36696d9cd970af2045211a7df/sandbox/processwrapper-sandbox/18693/execroot/com_github_cockroachdb_cockroach/bazel-out/s390x-fastbuild/bin/pkg/sql/opt/exec/execbuilder/tests/5node/5node_test_/5node_test.runfiles/com_github_cockroachdb_cockroach/pkg/sql/opt/exec/execbuilder/testdata/dist_vectorize with config 5node: 13 tests, 1 failures
    logic.go:4635: 
        /home/roach/.cache/bazel/_bazel_roach/c5a4e7d36696d9cd970af2045211a7df/sandbox/processwrapper-sandbox/18693/execroot/com_github_cockroachdb_cockroach/bazel-out/s390x-fastbuild/bin/pkg/sql/opt/exec/execbuilder/tests/5node/5node_test_/5node_test.runfiles/com_github_cockroachdb_cockroach/pkg/sql/opt/exec/execbuilder/testdata/dist_vectorize:191: error while processing
    logic.go:4635: /home/roach/.cache/bazel/_bazel_roach/c5a4e7d36696d9cd970af2045211a7df/sandbox/processwrapper-sandbox/18693/execroot/com_github_cockroachdb_cockroach/bazel-out/s390x-fastbuild/bin/pkg/sql/opt/exec/execbuilder/tests/5node/5node_test_/5node_test.runfiles/com_github_cockroachdb_cockroach/pkg/sql/opt/exec/execbuilder/testdata/dist_vectorize:191: too many errors encountered, skipping the rest of the input
    panic.go:615: -- test log scope end --
test logs left over in: /artifacts/tmp/_tmp/57a0b47a064ad2f0ca1cc34fcacc39f7/logTestExecBuild_dist_vectorize2879413257
--- FAIL: TestExecBuild_dist_vectorize (3.01s)
```
<details><summary>Help</summary>
<p>

See also: [How To Investigate a Go Test Failure \(internal\)](https://cockroachlabs.atlassian.net/l/c/HgfXfJgM)
</p>
</details>
<sub>

[This test on roachdash](https://roachdash.crdb.dev/?filter=status:open%20t:.*TestExecBuild_dist_vectorize.*&sort=title+created&display=lastcommented+project) | [Improve this report!](https://github.com/cockroachdb/cockroach/tree/master/pkg/cmd/bazci/githubpost/issues)

</sub>


Jira issue: CRDB-58436

pkg/sql/opt/exec/execbuilder/tests/5node/5node_test.TestExecBuild_dist_vectorize [failed](https://teamcity.cockroachdb.com/buildConfiguration/Cockroach_Ci_TestsIbmCloudLinuxS390x_UnitTestsS390x/20951917?buildTab=log) with [artifacts](https://teamcity.cockroachdb.com/buildConfiguration/Cockroach_Ci_TestsIbmCloudLinuxS390x_UnitTestsS390x/20951917?buildTab=artifacts#/) on master @ [ab117cbd0164f79e99d6e8668a1eb80cadf72b93](https://github.com/cockroachdb/cockroach/commits/ab117cbd0164f79e99d6e8668a1eb80cadf72b93):


```
            
        but found (query options: "") :
            planning time: 10µs
            execution time: 100µs
            distribution: <hidden>
            plan type: custom
            rows decoded from KV: 10 (80 B, 20 KVs, 10 gRPC calls)
            DistSQL network usage: <hidden>
            regions: <hidden>
            ·
            • merge join
            │ sql nodes: <hidden>
            │ regions: <hidden>
            │ execution time: 0µs
            │ actual row count: 5
            │ equality: (k) = (k)
            │ left cols are key
            │ right cols are key
            │
            ├── • scan
            │     sql nodes: <hidden>
            │     kv nodes: <hidden>
            │     regions: <hidden>
            │     KV time: 0µs
            │     KV rows decoded: 5
            │     actual row count: 5
            │     missing stats
            │     table: kv@kv_pkey
            │     spans: FULL SCAN
            │
            └── • scan
                  sql nodes: <hidden>
                  kv nodes: <hidden>
                  regions: <hidden>
                  KV time: 0µs
                  KV rows decoded: 5
                  actual row count: 5
                  missing stats
                  table: kw@kw_pkey
                  spans: FULL SCAN
            ·
            Diagram: https://cockroachdb.github.io/distsqlplan/decode.html#eJzsmt9u2sgXx-9_TzGaq_ZXUzz-Q4KlSmjTrpTuJqnSqjerqJrYJ2DZeKhngLBRHmtfYJ9sZRwnAWPKsYhNMFxEsT2253wO5zvfGeaOyp8hdejXT39-OvlG_k9-v7w4I8GEfL44PSfBlFyck2DyPiAfSDB9H1CNRsKDcz4ESZ2_KKMaNahGTapRi2rUplcaHcXCBSlFnDS5m99w6t1SR9eoH43GKjl9pVFXxECdO6p8FQJ16Dd-HcIlcA_itk416oHifjh_TTDpBZMfowBmVKMnIhwPI-mQQCMTqtGvI54ctdpJR_74TpL3SYdELD2Moe-LpIECqdJTyh-CQ_R__5EPTcRUEg9c4YHnkIf7rmcKJImBew45Jr-lJ_uXX06Iy8NQPrYbcT_O2iU9OPt-ckKkghFxxThS5A3cqrYfqbcO0edxpQ0AgqIGQ35LhjAU8YzwMBQuV0m39HkfrrlyByCJGKvRWDkkaT_vfnaC0at7jaZHKf-M7_WMDLgcLJLtJe2vNCoV7wN12LPsnX6kjn6vlUtgZymB014wXZXA6VMC2-yQwi2k0FhKYacwhU_PFbEHMXjLz32XdGSjViu-DWcQ9-Gz8COI22ypnkO4UW967N3bD7HfH6T_Uo1eJNH3mNYztJ6p9RJBgVtwx8oX0WLG1_FNrsmfIVEwHBHPlwEZS96H0vif0JpLaFlxeTxxGEeruK1Edi5aYtRm9iKsAgS_jMMujMNajsNeiINtXuYMr9Nto9U2D2W-baVmZVN4VEKpDyl8CaU-KkxhhUrNXpFSm5sr9WJ5GJuXh1FC4cxW2zqUx7YVziibwuMSCndI4Uso3HFhCitUOOMVKZy-ucItloe5eXmYJRTOarXtQ3lsW-HMsinsllC4QwpfQuG6hSmsUOHMV6RwiNn2YnlYm5eHVULh7NahOLatb1bZBNp4fWu1GeGRRxgRagDxIZnbVjq7MJkVKp31ipQO4eUszLL7JciRiCQsrS-ufpW-9KoWSxiD14c0c1KMYxe-xMKdt00PL-YPmtezB1KlV4304DTKLknF1boFyxdmm3QgBj58_Nli81A6haFEoKYiDkjIFUTu7DGW7PyU-2oxSg8kxD4P_b95HkF227zvMbjgTzJZyS5l2pJdS_UluzoEmXB53kDfsPSf82FIQN2aAelrAOlbAbT0BTKQgFhxNewpIRNLyG4aIStHiCEE1dgpQe3gQukUhrKvgooE1K0ZUPWCigTEiqthTwmZWEJ20whZOULGMiH9OSEzP-Jkj7Jz2myufRRbVDR9p8Q5F4tV0rjv6feG4fgUzwbqTzVDxtI8444EVLtxr9yqmFhCzTPuOUJ2SeO-p4COcHyKZwM7IKjIWJpn3JGAajfu1QsqllDzjHuOUAdr3CsFZK4BZFi_IsQ23Lq2MB_JDTlHO-3hKy8yA8en7hX96uc4SEDFE4P6x2QDGUvtHr76tSIsoeZ5-Byh45328JUL6jGOT90r-tULKhJQ8cRgBwQVGUvtHr56QcUSap6HzxHq7rSHrxyQnRtxWNkNNHtKyEQCqtvEV_8rMpZQ85b3sYTWzA3qH5dNdDTN8_F5RGV30ewpoS4SUN1OvnpVxRJq3ho_ltCaCcIOqCo6muaZ-Twi9Faa_RYNOz_u5HYINdvNW0hAzXPzWELNc_NYQg1c6UcjKp4i1D80W_loMHsVGzAy20hAzXPzWELNc_NYQg1c7kcjKp4i7ICq5qPJ7UDaKTdfvapa91cavQnF9IfvUYfqD5_Wij_ZhyY38L6kzh39OhDTOa1vsxFI6tzwUIJGz3gAH0FBPPQjXyrfpY6Kx3B__7__AgAA__-nAaOb
            
[19:41:19] --- done: /home/roach/.cache/bazel/_bazel_roach/c5a4e7d36696d9cd970af2045211a7df/sandbox/processwrapper-sandbox/5832/execroot/com_github_cockroachdb_cockroach/bazel-out/s390x-fastbuild/bin/pkg/sql/opt/exec/execbuilder/tests/5node/5node_test_/5node_test.runfiles/com_github_cockroachdb_cockroach/pkg/sql/opt/exec/execbuilder/testdata/dist_vectorize with config 5node: 13 tests, 1 failures
    logic.go:4635: 
        /home/roach/.cache/bazel/_bazel_roach/c5a4e7d36696d9cd970af2045211a7df/sandbox/processwrapper-sandbox/5832/execroot/com_github_cockroachdb_cockroach/bazel-out/s390x-fastbuild/bin/pkg/sql/opt/exec/execbuilder/tests/5node/5node_test_/5node_test.runfiles/com_github_cockroachdb_cockroach/pkg/sql/opt/exec/execbuilder/testdata/dist_vectorize:191: error while processing
    logic.go:4635: /home/roach/.cache/bazel/_bazel_roach/c5a4e7d36696d9cd970af2045211a7df/sandbox/processwrapper-sandbox/5832/execroot/com_github_cockroachdb_cockroach/bazel-out/s390x-fastbuild/bin/pkg/sql/opt/exec/execbuilder/tests/5node/5node_test_/5node_test.runfiles/com_github_cockroachdb_cockroach/pkg/sql/opt/exec/execbuilder/testdata/dist_vectorize:191: too many errors encountered, skipping the rest of the input
    panic.go:615: -- test log scope end --
test logs left over in: /artifacts/tmp/_tmp/57a0b47a064ad2f0ca1cc34fcacc39f7/logTestExecBuild_dist_vectorize3259250548
--- FAIL: TestExecBuild_dist_vectorize (3.54s)
```
<details><summary>Help</summary>
<p>

See also: [How To Investigate a Go Test Failure \(internal\)](https://cockroachlabs.atlassian.net/l/c/HgfXfJgM)
</p>
</details>
<sub>

[This test on roachdash](https://roachdash.crdb.dev/?filter=status:open%20t:.*TestExecBuild_dist_vectorize.*&sort=title+created&display=lastcommented+project) | [Improve this report!](https://github.com/cockroachdb/cockroach/tree/master/pkg/cmd/bazci/githubpost/issues)

</sub>

pkg/sql/opt/exec/execbuilder/tests/5node/5node_test.TestExecBuild_dist_vectorize [failed](https://teamcity.cockroachdb.com/buildConfiguration/Cockroach_Ci_TestsIbmCloudLinuxS390x_UnitTestsS390x/20959274?buildTab=log) with [artifacts](https://teamcity.cockroachdb.com/buildConfiguration/Cockroach_Ci_TestsIbmCloudLinuxS390x_UnitTestsS390x/20959274?buildTab=artifacts#/) on master @ [805e01250313cfa86aac4f2800ed6f03c8319589](https://github.com/cockroachdb/cockroach/commits/805e01250313cfa86aac4f2800ed6f03c8319589):


```
            
        but found (query options: "") :
            planning time: 10µs
            execution time: 100µs
            distribution: <hidden>
            plan type: custom
            rows decoded from KV: 10 (80 B, 20 KVs, 10 gRPC calls)
            DistSQL network usage: <hidden>
            regions: <hidden>
            ·
            • merge join
            │ sql nodes: <hidden>
            │ regions: <hidden>
            │ execution time: 0µs
            │ actual row count: 5
            │ equality: (k) = (k)
            │ left cols are key
            │ right cols are key
            │
            ├── • scan
            │     sql nodes: <hidden>
            │     kv nodes: <hidden>
            │     regions: <hidden>
            │     KV time: 0µs
            │     KV rows decoded: 5
            │     actual row count: 5
            │     missing stats
            │     table: kv@kv_pkey
            │     spans: FULL SCAN
            │
            └── • scan
                  sql nodes: <hidden>
                  kv nodes: <hidden>
                  regions: <hidden>
                  KV time: 0µs
                  KV rows decoded: 5
                  actual row count: 5
                  missing stats
                  table: kw@kw_pkey
                  spans: FULL SCAN
            ·
            Diagram: https://cockroachdb.github.io/distsqlplan/decode.html#eJzsmt9u2sgXx-9_TzGaq_ZXUzz-Q4KlSmjTrpTuJqnSqjerqJrYJ2DZeKhngLBRHmtfYJ9sZRwnAWPKsYhNMFxEsT2253wO5zvfGeaOyp8hdejXT39-OvlG_k9-v7w4I8GEfL44PSfBlFyck2DyPiAfSDB9H1CNRsKDcz4ESZ2_KKMaNahGTapRi2rUplcaHcXCBSlFnDS5m99w6t1SR9eoH43GKjl9pVFXxECdO6p8FQJ16Dd-HcIlcA_itk416oHifjh_TTDpBZMfowBmVKMnIhwPI-mQQCMTqtGvI54ctdpJR_74TpL3SYdELD2Moe-LpIECqdJTyh-CQ_R__5EPTcRUEg9c4YHnkIf7rmcKJImBew45Jr-lJ_uXX06Iy8NQPrYbcT_O2iU9OPt-ckKkghFxxThS5A3cqrYfqbcO0edxpQ0AgqIGQ35LhjAU8YzwMBQuV0m39HkfrrlyByCJGKvRWDkkaT_vfnaC0at7jaZHKf-M7_WMDLgcLJLtJe2vNCoV7wN12LPsnX6kjn6vlUtgZymB014wXZXA6VMC2-yQwi2k0FhKYacwhU_PFbEHMXjLz32XdGSjViu-DWcQ9-Gz8COI22ypnkO4UW967N3bD7HfH6T_Uo1eJNH3mNYztJ6p9RJBgVtwx8oX0WLG1_FNrsmfIVEwHBHPlwEZS96H0vif0JpLaFlxeTxxGEeruK1Edi5aYtRm9iKsAgS_jMMujMNajsNeiINtXuYMr9Nto9U2D2W-baVmZVN4VEKpDyl8CaU-KkxhhUrNXpFSm5sr9WJ5GJuXh1FC4cxW2zqUx7YVziibwuMSCndI4Uso3HFhCitUOOMVKZy-ucItloe5eXmYJRTOarXtQ3lsW-HMsinsllC4QwpfQuG6hSmsUOHMV6RwiNn2YnlYm5eHVULh7NahOLatb1bZBNp4fWu1GeGRRxgRagDxIZnbVjq7MJkVKp31ipQO4eUszLL7JciRiCQsrS-ufpW-9KoWSxiD14c0c1KMYxe-xMKdt00PL-YPmtezB1KlV4304DTKLknF1boFyxdmm3QgBj58_Nli81A6haFEoKYiDkjIFUTu7DGW7PyU-2oxSg8kxD4P_b95HkF227zvMbjgTzJZyS5l2pJdS_UluzoEmXB53kDfsPSf82FIQN2aAelrAOlbAbT0BTKQgFhxNewpIRNLyG4aIStHiCEE1dgpQe3gQukUhrKvgooE1K0ZUPWCigTEiqthTwmZWEJ20whZOULGMiH9OSEzP-Jkj7Jz2myufRRbVDR9p8Q5F4tV0rjv6feG4fgUzwbqTzVDxtI8444EVLtxr9yqmFhCzTPuOUJ2SeO-p4COcHyKZwM7IKjIWJpn3JGAajfu1QsqllDzjHuOUAdr3CsFZK4BZFi_IsQ23Lq2MB_JDTlHO-3hKy8yA8en7hX96uc4SEDFE4P6x2QDGUvtHr76tSIsoeZ5-Byh45328JUL6jGOT90r-tULKhJQ8cRgBwQVGUvtHr56QcUSap6HzxHq7rSHrxyQnRtxWNkNNHtKyEQCqtvEV_8rMpZQ85b3sYTWzA3qH5dNdDTN8_F5RGV30ewpoS4SUN1OvnpVxRJq3ho_ltCaCcIOqCo6muaZ-Twi9Faa_RYNOz_u5HYINdvNW0hAzXPzWELNc_NYQg1c6UcjKp4i1D80W_loMHsVGzAy20hAzXPzWELNc_NYQg1c7kcjKp4i7ICq5qPJ7UDaKTdfvapa91cavQnF9IfvUYfqD5_Wij_ZhyY38L6kzh39OhDTOa1vsxFI6tzwUIJGz3gAH0FBPPQjXyrfpY6Kx3B__7__AgAA__-nAaOb
            
[09:23:30] --- done: /home/roach/.cache/bazel/_bazel_roach/c5a4e7d36696d9cd970af2045211a7df/sandbox/processwrapper-sandbox/4630/execroot/com_github_cockroachdb_cockroach/bazel-out/s390x-fastbuild/bin/pkg/sql/opt/exec/execbuilder/tests/5node/5node_test_/5node_test.runfiles/com_github_cockroachdb_cockroach/pkg/sql/opt/exec/execbuilder/testdata/dist_vectorize with config 5node: 13 tests, 1 failures
    logic.go:4635: 
        /home/roach/.cache/bazel/_bazel_roach/c5a4e7d36696d9cd970af2045211a7df/sandbox/processwrapper-sandbox/4630/execroot/com_github_cockroachdb_cockroach/bazel-out/s390x-fastbuild/bin/pkg/sql/opt/exec/execbuilder/tests/5node/5node_test_/5node_test.runfiles/com_github_cockroachdb_cockroach/pkg/sql/opt/exec/execbuilder/testdata/dist_vectorize:191: error while processing
    logic.go:4635: /home/roach/.cache/bazel/_bazel_roach/c5a4e7d36696d9cd970af2045211a7df/sandbox/processwrapper-sandbox/4630/execroot/com_github_cockroachdb_cockroach/bazel-out/s390x-fastbuild/bin/pkg/sql/opt/exec/execbuilder/tests/5node/5node_test_/5node_test.runfiles/com_github_cockroachdb_cockroach/pkg/sql/opt/exec/execbuilder/testdata/dist_vectorize:191: too many errors encountered, skipping the rest of the input
    panic.go:615: -- test log scope end --
test logs left over in: /artifacts/tmp/_tmp/57a0b47a064ad2f0ca1cc34fcacc39f7/logTestExecBuild_dist_vectorize439621259
--- FAIL: TestExecBuild_dist_vectorize (3.84s)
```
<details><summary>Help</summary>
<p>

See also: [How To Investigate a Go Test Failure \(internal\)](https://cockroachlabs.atlassian.net/l/c/HgfXfJgM)
</p>
</details>
<sub>

[This test on roachdash](https://roachdash.crdb.dev/?filter=status:open%20t:.*TestExecBuild_dist_vectorize.*&sort=title+created&display=lastcommented+project) | [Improve this report!](https://github.com/cockroachdb/cockroach/tree/master/pkg/cmd/bazci/githubpost/issues)

</sub>

--- a/pkg/sql/opt/exec/execbuilder/testdata/dist_vectorize
+++ b/pkg/sql/opt/exec/execbuilder/testdata/dist_vectorize
@@ -135,23 +135,16 @@ EXPLAIN ANALYZE (DISTSQL) SELECT * FROM kv JOIN kw ON kv.k = kw.k
 planning time: 10µs
 execution time: 100µs
 distribution: <hidden>
-vectorized: <hidden>
 plan type: custom
 rows decoded from KV: 10 (80 B, 20 KVs, 10 gRPC calls)
-maximum memory usage: <hidden>
 DistSQL network usage: <hidden>
 regions: <hidden>
-isolation level: serializable
-priority: normal
-quality of service: regular
 ·
 • merge join
 │ sql nodes: <hidden>
 │ regions: <hidden>
-│ actual row count: 5
 │ execution time: 0µs
-│ estimated max memory allocated: 0 B
-│ estimated max sql temp disk usage: 0 B
+│ actual row count: 5
 │ equality: (k) = (k)
 │ left cols are key
 │ right cols are key
@@ -160,13 +153,9 @@ quality of service: regular
 │     sql nodes: <hidden>
 │     kv nodes: <hidden>
 │     regions: <hidden>
-│     actual row count: 5
 │     KV time: 0µs
 │     KV rows decoded: 5
-│     KV pairs read: 10
-│     KV bytes read: 40 B
-│     KV gRPC calls: 5
-│     estimated max memory allocated: 0 B
+│     actual row count: 5
 │     missing stats
 │     table: kv@kv_pkey
 │     spans: FULL SCAN
@@ -175,13 +164,9 @@ quality of service: regular
       sql nodes: <hidden>
       kv nodes: <hidden>
       regions: <hidden>
-      actual row count: 5
       KV time: 0µs
       KV rows decoded: 5
-      KV pairs read: 10
-      KV bytes read: 40 B
-      KV gRPC calls: 5
-      estimated max memory allocated: 0 B
+      actual row count: 5
       missing stats
       table: kw@kw_pkey
       spans: FULL SCAN

--- a/pkg/sql/opt/exec/execbuilder/testdata/explain_analyze_plans
+++ b/pkg/sql/opt/exec/execbuilder/testdata/explain_analyze_plans
@@ -123,31 +123,24 @@ EXPLAIN ANALYZE (DISTSQL) SELECT kv.k, avg(kw.k) FROM kv JOIN kw ON kv.k=kw.k GR
 planning time: 10µs
 execution time: 100µs
 distribution: <hidden>
-vectorized: <hidden>
 plan type: custom
 rows decoded from KV: 10 (80 B, 20 KVs, 10 gRPC calls)
-maximum memory usage: <hidden>
 DistSQL network usage: <hidden>
 regions: <hidden>
-isolation level: serializable
-priority: normal
-quality of service: regular
 ·
 • group (streaming)
 │ sql nodes: <hidden>
 │ regions: <hidden>
-│ actual row count: 5
 │ execution time: 0µs
+│ actual row count: 5
 │ group by: k
 │ ordered: +k
 │
 └── • merge join
     │ sql nodes: <hidden>
     │ regions: <hidden>
-    │ actual row count: 5
     │ execution time: 0µs
-    │ estimated max memory allocated: 0 B
-    │ estimated max sql temp disk usage: 0 B
+    │ actual row count: 5
     │ equality: (k) = (k)
     │ left cols are key
     │ right cols are key
@@ -156,13 +149,9 @@ quality of service: regular
     │     sql nodes: <hidden>
     │     kv nodes: <hidden>
     │     regions: <hidden>
-    │     actual row count: 5
     │     KV time: 0µs
     │     KV rows decoded: 5
-    │     KV pairs read: 10
-    │     KV bytes read: 40 B
-    │     KV gRPC calls: 5
-    │     estimated max memory allocated: 0 B
+    │     actual row count: 5
     │     missing stats
     │     table: kv@kv_pkey
     │     spans: FULL SCAN
@@ -171,13 +160,9 @@ quality of service: regular
           sql nodes: <hidden>
           kv nodes: <hidden>
           regions: <hidden>
-          actual row count: 5
           KV time: 0µs
           KV rows decoded: 5
-          KV pairs read: 10
-          KV bytes read: 40 B
-          KV gRPC calls: 5
-          estimated max memory allocated: 0 B
+          actual row count: 5
           missing stats
           table: kw@kw_pkey
           spans: FULL SCAN
@@ -251,38 +236,30 @@ EXPLAIN ANALYZE (DISTSQL) SELECT DISTINCT(kw.w) FROM kv JOIN kw ON kv.k = kw.w O
 planning time: 10µs
 execution time: 100µs
 distribution: <hidden>
-vectorized: <hidden>
 plan type: custom
 rows decoded from KV: 10 (80 B, 20 KVs, 10 gRPC calls)
-maximum memory usage: <hidden>
 DistSQL network usage: <hidden>
 regions: <hidden>
-isolation level: serializable
-priority: normal
-quality of service: regular
 ·
 • sort
 │ sql nodes: <hidden>
 │ regions: <hidden>
-│ actual row count: 5
 │ execution time: 0µs
-│ estimated max memory allocated: 0 B
+│ actual row count: 5
 │ order: +w
 │
 └── • distinct
     │ sql nodes: <hidden>
     │ regions: <hidden>
-    │ actual row count: 5
     │ execution time: 0µs
-    │ estimated max memory allocated: 0 B
+    │ actual row count: 5
     │ distinct on: w
     │
     └── • hash join
         │ sql nodes: <hidden>
         │ regions: <hidden>
-        │ actual row count: 5
         │ execution time: 0µs
-        │ estimated max memory allocated: 0 B
+        │ actual row count: 5
         │ equality: (k) = (w)
         │ left cols are key
         │
@@ -290,13 +267,9 @@ quality of service: regular
         │     sql nodes: <hidden>
         │     kv nodes: <hidden>
         │     regions: <hidden>
-        │     actual row count: 5
         │     KV time: 0µs
         │     KV rows decoded: 5
-        │     KV pairs read: 10
-        │     KV bytes read: 40 B
-        │     KV gRPC calls: 5
-        │     estimated max memory allocated: 0 B
+        │     actual row count: 5
         │     missing stats
         │     table: kv@kv_pkey
         │     spans: FULL SCAN
@@ -305,13 +278,9 @@ quality of service: regular
               sql nodes: <hidden>
               kv nodes: <hidden>
               regions: <hidden>
-              actual row count: 5
               KV time: 0µs
               KV rows decoded: 5
-              KV pairs read: 10
-              KV bytes read: 40 B
-              KV gRPC calls: 5
-              estimated max memory allocated: 0 B
+              actual row count: 5
               missing stats
               table: kw@kw_pkey
               spans: FULL SCAN

